core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -701,7 +701,7 @@ void processMeasurementEvent(WriterJob job) {
                             double v = Unsafe.getUnsafe().getDouble(bufPos);
                             bufPos += Double.BYTES;
                             final int colType = writer.getMetadata().getColumnType(colIndex);
-                            switch (colType) {
+                            switch (ColumnType.tagOf(colType)) {
```

Comment:
unnecessary, this is casting code

Reply:
still a switch, again let's think about it when ILP support is added for geohash